WebApi/WebApiTestts/LibraryManagerTest.cs

```diff
@@ -0,0 +1,139 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using WebApi.Library;
+using WebApi.Models;
+
+namespace WebApiTests
+{
+    [TestClass]
+    public class LibraryManagerTest
+    {
+        public static ILibraryManager library;
+
+        [ClassInitialize]
+        public static void TestInitialize(TestContext testContext)
+        {
+            library = new LibraryManager();
+        }
+
+        [TestMethod]
+        [DataRow(4, "Ray Bradbury", 1920)]
+        public void GetAuthor(int id, string name, int year)
```

Comment:
You should have unit tests for all positive and negative scenario. That's why you don't know that you have critical issues with your logic.